client/src/components/formHelpers/FormValidators.js

```diff
@@ -0,0 +1,29 @@
+// Matches editor links for: Repl.it, Glitch, CodeSandbox
+const editorRegex = /repl\.it\/@|edit\/#!|codesandbox\.io\/s\//;
+const localhostRegex = /localhost/;
+const IPv4 = new RegExp(
+  // eslint-disable-next-line max-len
+  /((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)/
+);
+const IPv6 = new RegExp(/((([0-9a-fA-F]){1,4})\\:){7}([0-9a-fA-F]){1,4}/);
+const validURLRegex = /^(https?:\/\/|www\.|localhost:\d+)/;
+
+const composeRegex = (...regexExpressions) => value =>
+  regexExpressions.some(exp => exp.test(value));
+
+export const editorValidator = value =>
+  editorRegex.test(value) ? 'Remember to submit the Live App URL.' : null;
+
+export const localhostValidator = value =>
+  localhostRegex.test(value)
+    ? 'Remember to submit a publicly visible app URL.'
+    : null;
+
+export const validURLValidator = value => {
```

Comment:
Are you thinking of adding this or is it for future proofing the validators?

Reply:
I added this, but then wondered:
a) I am not convinced with my RegEx
b) Is it at all useful

So, I left it to be up for discussion. Shall I remove them?

Reply:
I think we should remove them if we are not using them since the current form checks the validity of the urls to an extent. 

I will be using a library to validate urls in an upcoming pr, maybe we could incorporate instead of custom regex.